metric/push/no_cache/count

```diff
@@ -1,15 +1,17 @@
 # 10 people, 45000 messages total
-# Operation, Average Time (ms)
-# Commit hash: 15f2596f363566975b6f0540d92f7871191c1b94
+# Operation, Count
+# Commit hash: 305817756542bfb2ace6213431df5d6dd8c685cf
 
 client.logon	90.0
 client.p2pMsg	40725.0
 client.register	90.0
-clientServer.p2pMsg	44440.0
-db.messages.insertMany	44979.0
-db.messages.update	44979.0
-db.users.find	44979.0
+clientServer.p2pMsg	44339.0
```

Comment:
Seems something is wrong here??
When we run perf test all users are mostly online and ideally clientServer.p2pMsg should be close to 0 right, at least not as high as 44339?

Reply:
I feel we should rewrite perf test such that during perf test run:
- you start N clients and 1 server
- you tell each client whom it should send messages to and how many messages to send
- optionally add a few random logoff points for the client
- wait for all clients done
- shutdown server.